src/gui/torrentcontentmodel.cpp

```diff
@@ -77,24 +77,6 @@ namespace
     };
 
 #ifdef QBT_PIXMAP_CACHE_FOR_FILE_ICONS
```

Comment:
How about this define?

Reply:
what do you mean? I rather leave it be as it is related to file icons caching.

Reply:
>it is related to file icons caching.

Ok.

One more question. Is it defined by default? We have file icons performance related issues.

Reply:
It is switched on for windows & macos, the definition is just at the top of this file.